### ISSUE TITLE
Syncing bug?

### ISSUE THREAD

**Issue by johngarciava** [michaldrabik/showly-2.0#579]
So today I watched the latest episodes of Severance and The Problem with Jon Stewart and marked them as watched after I finished them on the app. However later on in the night I checked settings and noticed the app logged me out of the Trakt.tv authorization. So I log back in and force a sync (I make sure the 'export from Showly to Trakt.tv' is checked) between the app and trakt, however on Trakt I don't see the the episodes I had marked as watched through the app while I was logged out...Is this working as intended? What should I do here?

If this is not a bug, then I believe a pop-up or something that notifies the user that their trakt.tv authorization account has been logged out so I can log back in to have my marked episodes sync properly between trakt and showly.

Screenshot of the app showing I marked the Severance and Stewart shows as watched
![image](https://user-images.githubusercontent.com/29204882/158012049-d182cee2-80b6-47b1-9dfb-bd27406a278b.png)

Screenshot of my Trakt history after manually syncing multiple times to try to fix it (there is no severance or stewart which shouldve gone in the March 11, 2022 portion as I watched them before watching The Adam Project): 
![image](https://user-images.githubusercontent.com/29204882/158012058-71012ef7-8b20-42dd-b672-700a95f89dd9.png)

**Comment by michaldrabik** [comment]
Hey. Not sure what happens here just yet but definitely not a feature. 
Are you a long time user? How long ago did you first login before you noticed that app logged you out?
Were you using scheduled sync normally or always syncing manually?

**Comment by johngarciava** [comment]
I've only been using the app ever since December 2021, so like 4 months now.
I logged in basically the day I downloaded the app. This is the first time the app has ever logged me out. I haven't uninstalled the app or anything like that.
I use Instant Sync (Trakt.tv Instant Upload and Data Remove both turned on). The only times I sync manually (click sync now) is when I mark something from the Trakt website and I want the change to instantly appear on the app.

It's odd how I have the latest episodes of Severance and Stewart as watched in the app and yet when I click sync now to export that data from Showly to my Trakt while being logged in, the changes still don't update on Trakt on March 11 like it should.

**Comment by michaldrabik** [comment]
At this moment please try this:
1. Uncheck these episodes in the app.
2. Via Settings try manually loggin out and log in to Trakt again.
3. Make sure data remove and instant upload are active
4. Mark these episodes again - they should be sent to trakt fine.

Also I'd like to investigate logout myself. If possible please send me your app id so I can check logs.
It's located at the bottom of Settings screen.

Thanks

**Comment by johngarciava** [comment]
Sure, app id is 65dad1dc-b1dd

Before I do those steps, I'm wondering if I should actually go through with them? I ask because if I do those steps it's gonna mark those episodes as watched right now instead of yesterday, correct?

That being the case, if I want to keep the accuracy of my watch times, my best course of action would be to mark the episodes watched at the time I actually watched them on Trakt first (Because I can specifiy the exact time I watched an episode, can't do this with Showly), and then manually sync now on the app, which should theoretically work.

**Comment by michaldrabik** [comment]
Yep if you care about the time you should do this via Trakt.tv website.

**Comment by blackholeearth** [comment]
This happened to me couple of times. I have been  using showly since before 2021-03 

When im suspicious of app is logged out,  I goto comments try to write there. If i cant, I realize that app has logged-out from traktv.

**Comment by michaldrabik** [comment]
@blackholeearth Could you send me the app id (it's located at the bottom of Settings) as I'd like to check internal logs. Also, when you are getting logged out does the app show it in Settings screen? Or does it look like you are still logged in on the UI side?

**Comment by blackholeearth** [comment]
Since couple of weeks , showly says instant sync failed. All the time. Something is broken with the instant sync.
